spec/test-suite.json

```diff
@@ -0,0 +1,71 @@
+{
+    "suite":
+    {
+        "zmanim":
+        [
+            {
+                "setup":
+                {
+                    "calculator":
+                    {
+                        "name": "SunTimes",
```

Comment:
Are we planning on supporting tests for the NOAA calculator as well?  By all accounts, it appears to remain more accurate over the longer term.  CC @KosherJava

Reply:
@pinnymz, It would be my hope that we support both. It is my belief that the NOAA one is more accurate (but I have no way of proving it). It does deal with the concept of a year, and therefore has leap year correction built in. This in itself should make it more accurate in at least that area. There may very well be new implementations one day that are more accurate.

Reply:
The idea of having a name is that we can have multiple test suites. Will have a few files that have different test cases.